docs/contribute/release_checklist.md

```diff
@@ -11,9 +11,51 @@ These are instructions for preparing a release branch for Firefox Android and st
 **This part is 100% covered by the Release Management team. The dev team should not perform these steps.**
 
 1. Create a branch name with the format `releases_v[beta_version]` off of the `main` branch (for example, `releases_v109`) through the GitHub UI.
-`[beta_version]` should follow the Firefox Beta release number. See [Firefox Release Calendar](https://wiki.mozilla.org/Release_Management/Calendar).
-2. In `main` [version.txt](https://github.com/mozilla-mobile/firefox-android/blob/main/version.txt):
-   - Update the version from `[previous_nightly_version].0a1` to `[nightly_version].0a1`. For example:
+`[beta_version]` should follow the Firefox Beta release number. See [Firefox Release Calendar](https://wiki.mozilla.org/Release_Management/Calendar). The following below should be performed in this new beta release branch:
```

Comment:
The wiki page is being deprecated - replace the URL with https://whattrainisitnow.com/calendar/